File: Movie4U/Models/CountryModel.cs
using Movie4U.Entities;
using System.ComponentModel.DataAnnotations;

namespace Movie4U.Models
{
    public class CountryModel
    {
        public string country { get; set; }

        [Required, Key]
        public string country_code { get; set; }  // while getting from uNoGs, it's name might not contain '_' for this class 

        public int expiring { get; set; }

        public int nl7 { get; set; }

        public int tmovs { get; set; }

        public int tseries { get; set; }

        public int tvids { get; set; }

        public void Copy(Country source)
        {
            this.country = source.country;
            this.country_code = source.country_code;
            this.expiring = source.expiring;
            this.nl7 = source.nl7;
            this.tmovs = source.tmovs;
            this.tseries = source.tseries;
            this.tvids = source.tvids;
        }

    }
}

File: flutter_app/lib/models/country_model.dart
class CountryModel {
  CountryModel({
    required this.id,
    required this.country,
    required this.countrycode,
    required this.expiring,
    required this.nl7,
    required this.tmovs,
    required this.tseries,
    required this.tvids,
  });

  int id;
  String country;
  String countrycode;
  int expiring;
  int nl7;
  int tmovs;
  int tseries;
  int tvids;

  factory CountryModel.fromJson(Map<String, dynamic> json) => CountryModel(
        id: json["id"],
        country: json["country"],
        countrycode: json["countrycode"],
        expiring: json["expiring"],
        nl7: json["nl7"],
        tmovs: json["tmovs"],
        tseries: json["tseries"],
        tvids: json["tvids"],
      );

  Map<String, dynamic> toJson() => {
        "id": id,
        "country": country,
        "countrycode": countrycode,
        "expiring": expiring,
        "nl7": nl7,
        "tmovs": tmovs,
        "tseries": tseries,
        "tvids": tvids,
      };
}
